lib/diaspora_federation/entities/comment.rb

```diff
@@ -24,6 +24,12 @@ class Comment < Entity
       #   The timestamp when the comment was edited
       #   @return [Time] edited time
       property :edited_at, :timestamp, optional: true
+
+      # @!attribute [r] guid
+      #   A random string of at least 16 chars pointing to a parent comment.
+      #   @see Validation::Rule::Guid
+      #   @return [String] parent comment guid
+      property :thread_parent_guid, :string, optional: true
```

Comment:
The field should also be added to the `CommentValidator` to validate it's a valid GUID.
And it should also be added to the comment entity in `lib/diaspora_federation/schemas/federation_entities.json`.